plugins/flutter_aepconsent/pubspec.yaml

```diff
@@ -0,0 +1,37 @@
+name: flutter_aepconsent
```

Comment:
can we rename this to flutter_aepedgeconsent for uniformity with the other platforms? This applies for all the files/occurrences, except the dart class